src/app/integrations/edit-page/flow-view/flow-view-step.component.scss

```diff
@@ -86,20 +97,24 @@
         padding: 5px 0;
 
         li {
-          padding: 5px 5px 5px 17px;
+          padding: 5px 5px 5px 30px;
+
```

Comment:
rofl I can't bear it.. even in the `ts` files, but there's a lint rule for that, so it yells at me every time I try to keep the code separated and readable. :(